.github/scripts/toolchain/build-gcc.sh

```diff
@@ -12,52 +12,72 @@ if [ "$RUN_CONFIG" = 1 ] || [ ! -f "$GCC_BUILD_PATH/Makefile" ] ; then
 
     rm -rf $GCC_BUILD_PATH/*
 
-    case $TARGET in
+    case $PLATFORM in
         *mingw*)
-            HOST_OPTIONS=" \
+            TARGET_CONF=" \
                 --enable-threads=win32 \
                 --disable-win32-registry"
             ;;
+        *linux*)
+            TARGET_CONF=" \
+                --enable-threads=posix"
+            ;;
+    esac
+
+    case $ARCH in
+        x86_64)
+            TARGET_CONF="$TARGET_CONF \
+                --with-arch=nocona \
+                --with-tune=generic"
+            ;;
+        aarch64)
+            TARGET_CONF="$TARGET_CONF \
+                --with-arch=armv8-a \
+                --with-tune=cortex-a53"
+            ;;
     esac
 
     if [ "$DEBUG" = 1 ] ; then
-        ADDITIONAL_OPTIONS=" \
+        TARGET_CONF="$TARGET_CONF \
             --enable-debug"
     fi
 
+    # ADDED --includedir=$TOOLCHAIN_PATH/$TARGET/include
     # REMOVED --libexecdir=/opt/lib
-    # REMOVED --with-{gmp,mpfr,mpc,isl}=/usr
     $SOURCE_PATH/$GCC_VERSION/configure \
         --prefix=$TOOLCHAIN_PATH \
+        --build=$HOST \
+        --host=$HOST \
         --target=$TARGET \
         --includedir=$TOOLCHAIN_PATH/$TARGET/include \
+        --libexecdir=$TOOLCHAIN_PATH/lib \
+        --disable-bootstrap \
         --enable-languages=c,lto,c++,fortran \
         --enable-shared \
         --enable-static \
         --enable-graphite \
         --enable-fully-dynamic-string \
-        --enable-libstdcxx-filesystem-ts=yes \
-        --enable-libstdcxx-time=yes \
+        --enable-libstdcxx-filesystem-ts \
```

Comment:
The order has been changed to the same order as MSYS2 recipe has for better synchronization in the future.